hedera-node/hedera-app/build.gradle.kts

```diff
@@ -146,6 +138,6 @@ tasks.clean { dependsOn(cleanRun) }
 
 tasks.register("showHapiVersion") {
   doLast {
-    println(versionCatalogs.named("libs").findVersion("hapi-version").get().requiredVersion)
+    //    println(versionCatalogs.named("libs").findVersion("hapi-version").get().requiredVersion)
```

Comment:
In that case the complete task can be removed

Reply:
I think we can keep it back.